src/pub_sub/utils.ts

```diff
@@ -0,0 +1,67 @@
+import * as process from "process";
+
+export const NOTIFICATION_EVENT_TOPIC: string = "NOTIFICATION_EVENT_TOPIC2"
+export const NOTIFICATION_EVENT_GROUP: string = "NOTIFICATION_EVENT_GROUP"
+export const NOTIFICATION_EVENT_DURABLE: string = "NOTIFICATION_EVENT_DURABLE5"
+export const ORCHESTRATOR_STREAM: string = "ORCHESTRATOR_test678"
+const ackWait: number = parseInt(process.env.ACK_WAIT)
+const consumerReplica: number = parseInt(process.env.CONSUMER_REPLICAS)
+const maxAge: number = parseInt(process.env.MAX_AGE)
+const streamReplica: number = parseInt(process.env.STREAM_REPLICA)
+
+export interface NatsTopic {
```

Comment:
Please create types file for interfaces and constants file for constants. this file should only contain the utility functions